tests/test_layout.py

```diff
@@ -1212,6 +1212,14 @@ def test_focused_frame_child(hlwm):
         assert hlwm.get_attr('tags.0.tiling.focused_frame.index') == focused_index
 
 
+def verify_frame_indices(hlwm, root='tags.focus.tiling.root', index=[]):
+    path = root.rstrip('.') + '.' + '.'.join(index)
+    assert hlwm.get_attr(path.rstrip('.') + '.index') == ''.join(index)
+    if '0' in hlwm.list_children(path):
+        verify_frame_indices(hlwm, root=root, index=index + ['0'])
+        verify_frame_indices(hlwm, root=root, index=index + ['0'])
```

Comment:
Why is this called twice with the same arguments?

Reply:
Because it actually should be `'1'` in the second line. (The `if '0'`... only checks that it's a frame split).

Reply:
Thanks!